tests/test-verify-volume-mount.yml

```diff
@@ -53,6 +77,42 @@
       Found unexpected mount state for volume
       '{{ storage_test_volume.name }}' mount point
 
+- name: Verify mount directory user
+  assert:
+    that: "{{ storage_test_volume.mount_user ==
+              storage_test_found_mount_user.stdout }}"
+    msg: "Mount directory {{ storage_test_volume.mount_point }} of volume
+          {{ storage_test_volume.name }}) has unexcepted owner
+          (expected: {{ storage_test_volume.mount_user }}, found:
+          {{ storage_test_found_mount_user.stdout }})"
+  when: _storage_test_volume_present and
+        storage_test_volume.mount_point and
+        storage_test_volume.mount_user
+
+- name: Verify mount directory group
+  assert:
+    that: "{{ storage_test_volume.mount_group ==
```

Comment:
see above - note that in an Ansible conditional context, such as a `when`, `that`, or similar, Ansible is already expecting that the value is a valid Jinja expression - it is not recommended to use `"{{ jinja expression }}"` - just use `jinja expression`

Reply:
Fixed.